spec/models/period_contribution_calculator_spec.rb

```diff
@@ -24,6 +25,10 @@
       let(:employee_percent) { 1 }
       let(:employer_percent) { 1 }
 
+      it 'knows which period its calculating contributions' do
+        expect(period_contribution.name).to eq 'some period'
```

Comment:
The conversations around commas have not been resolved so I'll leave this one alone :)